lua/blink/compat/init.lua

```diff
@@ -26,7 +26,12 @@ local function setup_events()
   local function make_entry(item)
     -- NOTE: only events emmited by blink.compat sources will have a `source`
     return {
-      entry = { completion_item = item, source = item._source or {} },
+      entry = {
+        completion_item = item,
+        source = item._source or {},
+        get_commit_characters = function() return '' end,
```

Comment:
Should this return an empty table instead? See [nvim-cmp](https://github.com/hrsh7th/nvim-cmp/blob/f17d9b4394027ff4442b298398dfcaab97e40c4f/lua/cmp/entry.lua#L376)

Reply:
oh, based on this it definitely should. for autopairs it was enough that it existed since it only reads the value, but never uses, so I just took a guess, what it can be